joomla/joomla-cms
# [4.0] Remove com_admin profile view

Pull Request for Issue # .

### Summary of Changes
The PR https://github.com/joomla/joomla-cms/pull/32771 was merged, so view profile in com_admin is not used anymore, thus it could be removed.


### Testing Instructions
As it is not used, I don't know what should be testing instructions. Could it be merged on review @wilsonge ?

Or 

1. Enable at least one 2fa plugin
2. Setup "enforce 2fa" within the Users -> Manage -> Options
3. Make sure that you are redirected to edit user account page (before it's com_admin but now it is com_users)
